README.md

```diff
@@ -5,6 +5,11 @@ Shunter is a [Node.js][node] application built to read JSON and translate it int
 
 It helps you create a loosely-coupled front end which can serve traffic from one or more back end applications - great for use in multi-language, multi-disciplinary teams or just to make your project more flexible and future-proofed.
 
+Shunter does not contain an API client, or any Controller logic (in the [MVC](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller) sense). Instead, Shunter simply proxies requests to a back end server, then:
+
+1. If the back end wants Shunter to render the response, it returns the application state as JSON, served with a certain HTTP header. This initiates the templating process in Shunter.
+2. If the back end wishes to serve the response, it omits the header and Shunter proxies the request back to the client.
```

Comment:
Is this worth suffixing with something such as: "This is often used to serve resources such as images and CSS"?

Reply:
mmmhmm could do, but don't want to muddy the water at this early stage in the docs as shunter can also serve assets...